build.sbt

```diff
@@ -58,13 +56,7 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
 )
 scalastyleConfig := baseDirectory.value / "project" / "scalastyle-config.xml"
 
-lazy val silencerSettings: Seq[Setting[_]] = {
```

Comment:
why have we removed this? Do we not need it

Reply:
The guide mentions we no longer need it. If we want to suppress warnings we should use scalacOptions instead